diff --git a/src/SetLocale.c b/src/SetLocale.c
index 6730338..2302c44 100644
--- a/src/SetLocale.c
+++ b/src/SetLocale.c
@@ -121,7 +121,7 @@ _Xsetlocale(
 
 #else /* X_LOCALE */
 
-#ifdef __DARWIN__
+#ifdef __APPLE__
 char *
 _Xsetlocale(
     int           category,
@@ -130,7 +130,7 @@ _Xsetlocale(
 {
     return setlocale(category, name);
 }
-#endif /* __DARWIN__ */
+#endif /* __APPLE__ */
 
 /*
  * _XlcMapOSLocaleName is an implementation dependent routine that derives
diff --git a/src/xlibi18n/XlcPubI.h b/src/xlibi18n/XlcPubI.h
index 2b4b30f..4544fb8 100644
--- a/src/xlibi18n/XlcPubI.h
+++ b/src/xlibi18n/XlcPubI.h
@@ -240,7 +240,7 @@ _Xsetlocale(
     int           category,
     _Xconst char  *name);
 #else
-#ifdef __DARWIN__
+#ifdef __APPLE__
 extern char *
 _Xsetlocale(
     int           category,
